lang/udf_cache.hh

```diff
@@ -110,7 +110,17 @@ public:
 private:
     value_type load(wasm::context& ctx) {
         auto store = wasmtime::Store(ctx.engine_ptr->get());
-        auto instance_res = wasmtime::Instance::create(store, *ctx.module, {});
+        auto linker = wasmtime::Linker(ctx.engine_ptr->get());
+        auto wasi_def = linker.define_wasi();
```

Comment:
Should this be merged into a previous patch? We don't introduce problems in an early patch of a series to fix it later in the same series.